packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -194,6 +196,8 @@ export interface CallAdapterCallManagement {
     startScreenShare(): Promise<void>;
     stopCamera(): Promise<void>;
     stopScreenShare(): Promise<void>;
+    // @beta
+    toggleHold(): Promise<void>;
```

Comment:
is there a strong case for `toggle` here? Are other handlers are in pairs of stop/start so for consistency might be better to have `hold()` and `endHold()` (not convinced `endhold` is the right terminology lol)

Reply:
Resume() perhaps?

Reply:
Will do!